markdown/cmark/Makefile.am

```diff
@@ -0,0 +1,43 @@
+if MD_BUILTIN_CMARK
+
+noinst_LTLIBRARIES = libcmark.la
+
+libcmark_la_SOURCES = \
+	blocks.c \
+	buffer.c \
+	buffer.h \
+	chunk.h \
+	cmark.c \
+	cmark_ctype.c \
+	cmark_ctype.h \
+	cmark.h \
```

Comment:
Good call on the `cmark_version.h` missing. I tried to run `make distcheck` but it failed in what seemed like not my plugin's code (in `util` IIRC), will have to try again.

Will look at generating `cmark_version.h`, I was too lazy to bother as the changes I made are quite trivial to reproduce if/when I update the embedded version, but it is better to generate it. I briefly looked at `cmark_export.h.in` but I couldn't really figure out where it came from, and I can't read the awful `cmake` language.

Reply:
About distcheck failure in *util* I don't know but could believe it, I'm afraid nobody tried it since util was introduced.  I'll try to get a look into this when I can.

> I briefly looked at `cmark_export.h.in` but I couldn't really figure out where it came from

Isn't the *.in* the source itself?  Usually it's the suffix for files that will be processed, and the suffix is stripped from the output filename.

Reply:
Sorry, I made a typo with the `.in` suffix, it's just `cmark_export.h` and I have no clue where it comes from, it's not [in the sources](https://github.com/commonmark/cmark/tree/0.28.3/src), I can only suppose that it's entirely generated from CMake, but I couldn't stomach reading that horrid language.